src/components/Calculator.css

```diff
@@ -69,11 +77,25 @@ body {
 
 .container {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   justify-content: center;
   align-items: center;
 }
 
+h2 {
+  color: black;
+  font-size: 6vw;
+  width: 100%;
+  text-align: center;
+}
+
+.calciText h2 {
+  color: aliceblue;
+  font-size: 3vw;
+  width: 100%;
+  text-align: center;
+}
+
```

Comment:
- [ ] Kindly make sure to align correctly the main title on the homepage as it looks on the provided wireframe designs.

    ## YOUR DESIGN
   ![chrome-capture-2023-8-21](https://github.com/bhushan354/Math-Magician/assets/87197412/87f34069-468d-4dee-bc85-4a754235a4a1)

    ## PROVIDED DESIGN

    ![chrome-capture-2023-8-21 (1)](https://github.com/bhushan354/Math-Magician/assets/87197412/b39ad79b-a085-40ec-8398-ec2fd69b3569)